File: ticketdesk.client/src/model/CustomerDTO.ts
import { UserDTO } from "./UserDTO";

export class CustomerDTO {
  customerId!: string | null;
  user!: UserDTO;
  createdOn!: Date | null;
  createdBy!: string | null;
  modifiedOn!: Date | null;
  modifiedBy!: string | null;


}

File: TicketDesk.DTO/Customer/CustomerDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TicketDesk.DTO.User;

namespace TicketDesk.DTO.Customer
{
    public class CustomerDTO
    {
        public Guid? CustomerId { get; set; }
        public UserDTO User { get; set; }
        public DateTime? CreatedOn { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
    }
}
